app/src/main/java/org/mozilla/rocket/content/travel/data/TravelRepository.kt

```diff
@@ -29,14 +31,17 @@ class TravelRepository(
     }
 
     suspend fun getCityWiki(name: String): Result<Wiki> {
-        val resultExtract = remoteDataSource.getCityWikiExtract(name)
-        val resultImage = remoteDataSource.getCityWikiImage(name)
+        val resultName = remoteDataSource.getCityWikiName(name)
```

Comment:
I think we should also keep repository as simple as possible. It should be use case's responsibility(domain knowledge) to coordinate the call sequences to get the composite result. However, it's not a blocker now, please help to refactor it later. Thanks!